subprojects/core/src/integTest/groovy/org/gradle/api/internal/buildevents/BuildResultLoggerFunctionalSpec.groovy

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.buildevents
+
+import org.fusesource.jansi.Ansi
+import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
+
+class BuildResultLoggerFunctionalSpec extends AbstractConsoleFunctionalSpec {
+
+    def setup() {
+        executer.withStackTraceChecksDisabled()
+    }
+
+    def "Failure status is logged even in --quiet"() {
```

Comment:
Testing at a higher level and the color specifically.  But yes, there is definitely duplication.  After we get sufficient test coverage at the functional level it may make sense to remove some duplicate tests.